### PR TITLE
[meshoptimizer] Update to 0.16

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates meshoptimizer to 0.16

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  As before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run ./vcpkg x-add-version --all and committed the result?  
  Yes

### CODE REVIEW

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/meshoptimizer/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,16 @@
 {
   "name": "meshoptimizer",
-  "version-string": "0.14",
-  "port-version": 2,
+  "version-string": "0.16",
```
Comment (NancyLi1013) [inline ports/meshoptimizer/vcpkg.json:3]:
```suggestion
  "version": "0.16",
```
This might be served by `version`.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/meshoptimizer/portfile.cmake [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/meshoptimizer)
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENSE.md
-    DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT}
-    RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/LICENSE.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
 
 vcpkg_copy_pdbs()
+
```
[the remark this answers is not in the document]
Comment (chausner) [reply ports/meshoptimizer/portfile.cmake]: Addressed in 3a728e9fab18a133f92a82a3d6e1218061399428.

File: ports/meshoptimizer/vcpkg.json (hunk 1 above)
Comment (chausner) [reply ports/meshoptimizer/vcpkg.json]: Addressed in 3a728e9fab18a133f92a82a3d6e1218061399428.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM, thanks for your updates @chausner.

**Comment by BillyONeal** [comment]
Thanks again for the updates!

**Pull request merged**